lib/pry/prompt.rb

```diff
@@ -109,14 +108,16 @@ def prompt_name(name)
       )
     end
 
-    add(:simple, "A simple `>>`.\n", ['>> ', ' | ']) do |_, _, _, sep|
+    add 'simple',
```

Comment:
Is highlighting also broken in this example?

Reply:
nope, this works fine, i changed it to be consistent with the other cases using `add`.